### ISSUE TITLE
Unable to remove the only online account I have

### ISSUE THREAD

**Issue by persello** [elementary/switchboard-plug-onlineaccounts#137]
Even logging out and back in or rebooting the device does not solve the issue.
I attach a short recording of what happens (don't worry about the address, it's the one I use on GitHub).

![Screen record from 2020-05-24 17 01 58](https://user-images.githubusercontent.com/45900950/82757562-830b9c00-9de1-11ea-91e9-2fb02a7db1f2.gif)

**Comment by yerbestpal** [comment]
I am not actually able to remove **any** accounts.

**Comment by lowski** [comment]
@persello @yerbestpal I had the [very similar issue](https://www.reddit.com/r/elementaryos/comments/khyahm/unable_to_remove_online_account/) and was able to fix it by removing following directories:

- `~/.config/autostart/pantheon-mail-autostart.desktop`
- `~/.config/libaccounts-glib`

I'm not fully sure what else is stored in `~/.config/libaccounts-glib` so I recommend to backup it first.

**Comment by riker09** [comment]
I have had a look inside the SQlite database in `~/.config/libaccounts-glib/accounts.db` (the only file inside that folder, if you don't count the zero-sized journal file). It holds the account I created, but only the name. no configuration, no password, nothing. I was looking for a way to change the port I had entered when initially creating the account. But there is no way to do this via the GUI (or at least none that I could find) and the database is empty as well?
